### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by TumiRantsane**
Add mobile menu using JavaScript in the personal portfolio project.

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @TumiRantsane, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Ouch-gifs-38715704-365-199


You have submitted a project that has an incorrect linters configuration. Kindly update `./github/workflows/linters.yml` using [this](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) configuration from linters setup for [HTML, CSS and JavaScript](https://github.com/microverseinc/linters-config/tree/master/html-css-js) projects.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by prolajumokeoni** [comment]
Hi @TumiRantsane 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pair-programming project, that does not have commit from your partner and  Kindly use the updated version  of the microverse [readme](https://github.com/microverseinc/readme-template) template for your readme, this is a requirement to get a review, just as the previous reviewer requested please set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters

this was missed by the previous reviewer 👍🏿

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @TumiRantsane, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Ouch-gifs-38715704-365-199

You have made a great effort adjusting your readme using the new template. However, as the previous reviewer had mentioned, this is a pair programming project and commits are expected from all members of the team as an evidence of their contribution.

If you have issues connecting with your partner(s), kindly reach out to the student success team and update the PR description with an email received from them approving that you work alone.

Note that it is quite unprofessional to delete a reviewer's comment. I kindly suggest that you avoid doing so.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -14,6 +14,373 @@
...
+              </li>
+              <li class="navbar__menu__li ">
+                <a
+                  href="#contact"
+                  class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600"
+                  >Contact</a
+                >
+              </li>
+            </ul>
+          </div>
```
Comment (thecodechaser) [inline index.html:47]: - [ ] Please apply the bluer effect on the background to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Chuai4vEE9yvqNFx-0) design.
  - For filter, you can use `filter: blur(Npx);`. N is an integer number for blur capacity.

### Your design
image

### Figma design
image

File: index.html [hunk 2]
```diff
@@ -14,6 +14,373 @@
...
+          <img
+            class="navbar__hamb "
+            src="images/hamburger.png"
+            alt="hamburger icon"
+          />
+        </nav>
```
Comment (thecodechaser) [inline index.html:53]: - [ ] When I close the mobile menu, The hamburger icon is broken, Please fix this issue.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @TumiRantsane, and Team👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @TumiRantsane   , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ] Good Job using new  [README template](https://github.com/microverseinc/readme-template), Kindly customize according to your Project. 

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -14,6 +14,373 @@
   <title>Itumeleng Rantsane | Portfolio </title>
 </head>
 
+<<<<<<< HEAD
```
Comment (HAMMAS-SALEEM) [inline index.html:17]: Capture

- [ ] Kindly remove that **<<<<<< Head** shown in the image above ⬆️to make your project look professional as suggested by the previous reviewer 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @danielsafari143 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the reqested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Good overall design ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
# STATUS: INVALID REQUEST

Hi @danielsafari143 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### As requested in the project requirements:
image
### You should create the mobile menu on your project as well 😉.

Your Code Review Request will be marked invalid in your Dashboard, so please submit a new one once ready 🙏.
Invalid requests don't count as a review, so don't worry about that 😄!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**